src/math/vec2.js

```diff
@@ -164,6 +185,74 @@ class Vec2 {
         return Math.sqrt(x * x + y * y);
     }
 
+    /**
+     * @function
+     * @name Vec2#div
+     * @description Divides a 2-dimensional vector by another in place.
+     * @param {Vec2} rhs - The vector to divide the specified vector by.
+     * @returns {Vec2} Self for chaining.
+     * @example
+     * var a = new pc.Vec2(4, 9);
+     * var b = new pc.Vec2(2, 3);
+     *
+     * a.div(b);
+     *
+     * // Should output [2, 3]
+     * console.log("The result of the division is: " + a.toString());
+     */
+    div(rhs) {
+        this.x /= rhs.x;
+        this.y /= rhs.y;
+
+        return this;
+    }
+
+    /**
+     * @function
+     * @name Vec2#div2
+     * @description Divides one 2-dimensional vector by another and writes the result to
+     * the specified vector.
+     * @param {Vec2} lhs - The dividend vector (the vector being divided).
+     * @param {Vec2} rhs - The divisor vector (the vector dividing the dividend).
+     * @returns {Vec2} Self for chaining.
+     * @example
+     * var a = new pc.Vec2(4, 9);
+     * var b = new pc.Vec2(2, 3);
+     * var r = new pc.Vec2();
+     *
+     * r.add2(a, b);

```

Comment:
div2

Reply:
For these kinds of changes, please use GitHub's `suggestion` feature, so I can quickly merge them. 😄